script/travis_run_js_tests.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 set -ev
-if [[ $( git diff --name-only HEAD~1..HEAD config/webpack.config.js webpack/ .travis.yml package.json | wc -l ) -ne 0 ]]; then
+if [[ $( git diff --name-only origin/develop..HEAD config/webpack.config.js webpack/ .travis.yml package.json | wc -l ) -ne 0 ]]; then
```

Comment:
changed so the comparison will be also on multiple commits and not just the latest PR commit